lib/skc_ast2hir/src/convert_exprs/method_call.rs

```diff
@@ -162,6 +166,18 @@ pub fn arrange_named_args<'a>(
     Ok(v)
 }
 
+/// Check if number of arguments matches to the params.
+fn validate_argument_length(total_args: usize, params: &[MethodParam]) -> Result<()> {
+    if total_args > params.len() {
```

Comment:
[NOTE]
I want to validate fewer argument also but after make this comparison to `total_args != params.len()`, the change breaks tests.

I'll work for validating fewer argument error in next issue if you like it.
In that case, please notify me and assign me.

Reply:
Screen Shot
<img width="831" alt="image" src="https://github.com/shiika-lang/shiika/assets/86505801/9c330e80-9a7b-4aca-915a-79b491ac6ad9">

Reply:
> I want to validate fewer argument also but after make this comparison to total_args != params.len(), the change breaks tests.

Yes, that's because a parameter can have a default value.

```sk
class A
  def self.foo(a: Int, b = 2); end
end
A.foo(1)
```

Reply:
Ahhhhh I got it.